packages/manager/src/components/ErrorState/ErrorState.tsx

```diff
@@ -53,7 +54,12 @@ const ErrorState = (props: Props) => {
         <div className={classes.iconContainer}>
           {CustomIcon ? (
             <CustomIcon
-              className={classes.icon}
+              className={classNames(
+                {
+                  [classes.icon]: true,
+                },
+                className
+              )}
```

Comment:
I think this can be simplified

```suggestion
              className={classNames(classes.icon, className)}
```